.github/workflows/run-tests.yml

```diff
@@ -10,16 +10,22 @@ jobs:
       fail-fast: false
       matrix:
         php: [8.3, 8.2, 8.1, 8.0]
-        laravel: ["^10.0", "^9.0", "^8.12"]
+        laravel: ["^11.0", "^10.0", "^9.0", "^8.12"]
         dependency-version: [prefer-lowest, prefer-stable]
         include:
+          - laravel: "^11.0"
+            testbench: 9.*
           - laravel: "^10.0"
             testbench: 8.*
           - laravel: "^9.0"
             testbench: 7.*
           - laravel: "^8.12"
             testbench: "^6.23"
         exclude:
+            - laravel: "^11.0"
+              php: 8.1
```

Comment:
Laravel 11 requires php 8.2

Reply:
> Laravel 11 requires php 8.2

Right. That's why the `exclude` section of the test suite tells it to exclude testing against PHP 8.1.